### ISSUE TITLE
postponing iptv simple restart

### ISSUE THREAD

**Issue by polojoe** [add-ons/service.iptv.manager#84]
Could you please add option to prevent iptv simple to restart if there's ongoing playback of a recording (vdr/tvh)?

**Comment by dagwieers** [comment]
AFAIK IPTV Manager already does this.
https://github.com/add-ons/service.iptv.manager/blob/master/resources/lib/modules/iptvsimple.py#L73

**Comment by polojoe** [comment]
It works for livetv but not for playback recordings. So playback gets aborted.

**Comment by michaelarnauts** [comment]
Hmm, I think we can also exclude restarting of simple IPTV when playing any video file, but it's odd that a restart of IPTV simple stops a playback of a recording, since recordings aren't managed through IPTV simple.

You mentioned tvh, so you are playing a recording from Tvheadend?

**Comment by dagwieers** [comment]
That is strange. We have never seen this and we sync IPTV Manager every 2 hours. Running Kodi v18 on LibreELEC on RPi.

**Comment by polojoe** [comment]
Yes, playback of recording gets aborted. I tried tvheadend and vdr, no difference. It took me some time to find the cause. I'm running Kodi v19 with libreelec on rpi 4.

**Comment by RKCRLR** [comment]
I posted issue #89 but this problem sounds similar.  Playback of recordings or Pluto.TV gets aborted when the guide gets refreshed on my Shield TV.
